Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,114 @@
+"""
+NCL_sat_1.py
+===============
+This script illustrates the following concepts:
+   - Drawing line contours over a satellite map
+   - Manually labeling contours
+   - Transforming coordinates
```

Comment:
I would add that this example illustrates how to draw an orthographic map. If someone is trying to google for and example like this, adding that comment should make it easier for them to find it.